### ISSUE TITLE
ERROR: No module named 'base' on Mac High Sierra, LO 6.0.5

### ISSUE THREAD

**Issue by zopyx** [dagwieers/unoconv#456]
LibreOffice 6.0.5 on Mac,
current unoconv checkout:

```
../bin/unoconv
unoconv: Cannot find a suitable pyuno library and python binary combination in /Applications/LibreOffice.app/Contents
ERROR: No module named 'base'

unoconv: Cannot find a suitable office installation on your system.
ERROR: Please locate your office installation and send your feedback to:
       http://github.com/dagwieers/unoconv/issues
```

**Comment by douglasjunior** [comment]
Similar here:
```
unoconv: Cannot find a suitable pyuno library and python binary combination in /Applications/LibreOffice.app/Contents
ERROR: No module named uno

unoconv: Cannot find a suitable office installation on your system.
ERROR: Please locate your office installation and send your feedback to:
       http://github.com/dagwieers/unoconv/issues
```

**Comment by regebro** [comment]
I've noticed that when these issues arise, the solution is to run unoconv with the python executable that comes with the Libreoffice installation you want to use. Like so:

```
UNOPATH=/path/to/Libreoffice /path/to/Libreoffice/program/python /usr/bin/unoconv ....
```
Yes, that's fairly annoying, but only happens with multiple LibreOffice and/or Python installations.

Closed as duplicate of #49 .
